newscout_web/advertising/urls.py

```diff
@@ -8,19 +8,24 @@
                     CampaignDeleteView, AdGroupView, AdGroupDeleteView,
                     AdvertisementView, AdvertisementDeleteView,
                     CampaignCategoriesListView, GroupTypeListView,
-                    AdRedirectView)
+                    AdRedirectView, AdGroupListView, CampaignListView,
+                    AdvertisementListView)
 
 urlpatterns = [
     url(r'^schedules/$', GetAds.as_view(), name='get-ads'),
     url(r'campaigns/categories/$', CampaignCategoriesListView.as_view(),
         name='campign-categories-view'),
+    url(r'campaigns/list/$', CampaignListView.as_view(), name='campign-list'),
```

Comment:
Try and marge all `*/list` with their respective APIs. Why do this because of two reasons:

1. We don't need to create an extra end point
1. REST Semantics of using GET for listing would be solved. 

The way it would work is as follows

1. If a GET request is made using an `ID` information for that specific instance will get returned
1. If NO `ID` is given it will return a list, this should also support Pagination currently implemented by `*/list` endpoints

Reply:
fixed in commit id: 72f358238c9f9fc26ded0f3329cd92812c79660f